docs/getting-started/policies.md

```diff
@@ -838,3 +839,27 @@ The syntax:
    }
 }
 ```
+
+## CronTrigger Rule
+
+Cron trigger rule allows to set the minimum interval between the process triggered by cron.
```

Comment:
> allows to

There's a word or phrase missing here...one of `you`, `administrators`, `the ability`

Reply:
updated